examples/MultiCanvas.cpp

```diff
@@ -111,24 +119,115 @@ void runSw()
+    }
+};
+#endif
+
 void runGl()
 {
-#if 1
-    cout << "Not Support OpenGL" << endl;
-#else //TODO with FBO Target?
+#ifdef THORVG_GL_RASTER_SUPPORT
+
+#ifdef THORVG_GL_TARGET_GLES
+        SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_ES);
+        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
+        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 0);
+#else
+        SDL_GL_SetAttribute(SDL_GL_CONTEXT_PROFILE_MASK, SDL_GL_CONTEXT_PROFILE_CORE);
+        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
+        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 3);
+#endif
+
     auto window = SDL_CreateWindow("ThorVG Example (OpenGL)", SDL_WINDOWPOS_UNDEFINED, SDL_WINDOWPOS_UNDEFINED, WIDTH, HEIGHT, SDL_WINDOW_OPENGL | SDL_WINDOW_HIDDEN);
     auto context = SDL_GL_CreateContext(window);
 
+    fglBindTexture = (PFNGLBINDTEXTUREEXTPROC)SDL_GL_GetProcAddress("glBindTexture");
+    fglDeleteTextures = (PFNGLDELETETEXTURESEXTPROC)SDL_GL_GetProcAddress("glDeleteTextures");
+    fglGenTextures = (PFNGLGENTEXTURESEXTPROC)SDL_GL_GetProcAddress("glGenTextures");
+    fglTexImage2D = (PFNGLTEXIMAGE2DPROC)SDL_GL_GetProcAddress("glTexImage2D");
+    fglTexParameteri = (PFNGLTEXPARAMETERIPROC)SDL_GL_GetProcAddress("glTexParameteri");
+    fglBindFramebuffer = (PFNGLBINDFRAMEBUFFERPROC)SDL_GL_GetProcAddress("glBindFramebuffer");
+    fglGenFramebuffers = (PFNGLGENFRAMEBUFFERSPROC)SDL_GL_GetProcAddress("glGenFramebuffers");
+    fglDeleteFramebuffers = (PFNGLDELETEFRAMEBUFFERSPROC)SDL_GL_GetProcAddress("glDeleteFramebuffers");
+    fglFramebufferTexture2D = (PFNGLFRAMEBUFFERTEXTURE2DPROC)SDL_GL_GetProcAddress("glFramebufferTexture2D");
+    fglBlitFramebuffer = (PFNGLBLITFRAMEBUFFERPROC)SDL_GL_GetProcAddress("glBlitFramebuffer");
+
+    // Create the framebuffer which the GlCanvas can render to
+    // Since the example is just a simple demo, and only run the rendering function once
+    // we can just create one framebuffer
+    GLFrameBuffer glFbo{SIZE, SIZE};
+    
     for (int counter = 0; counter < NUM_PER_LINE * NUM_PER_LINE; ++counter) {
         auto canvas = tvg::GlCanvas::gen();
 
-        //Set the canvas target and draw on it.
-        tvgexam::verify(canvas->target(0, WIDTH, HEIGHT));
+        // Pass the framebuffer id to the GlCanvas
+        tvgexam::verify(canvas->target(glFbo.fbo, SIZE, SIZE));
 
         content(canvas.get());
         if (tvgexam::verify(canvas->draw())) {
             tvgexam::verify(canvas->sync());
         }
+
+        // After the GlCanvas::sync() function, the content is rendered to the framebuffer
+        // The texture is now ready to be blit to the screen
+        auto y = (counter / NUM_PER_LINE) * SIZE;
+        auto x = (counter % NUM_PER_LINE) * SIZE;
+        glFbo.blitToScreen(x, y, SIZE, SIZE);
+
+        // After the framebuffer is blit to the screen, the framebuffer and texture can be reused in the next iteration
     }
 
     SDL_ShowWindow(window);
```

Comment:
@RuiwenTang Relocated ShowWindow() call from line 196. This prevents the blank screen for a split second at the beginning. Please let me know if you have any opinions.

Reply:
Look good to me :)